models/base/BCEfficiencyFitter.cxx

```diff
@@ -28,29 +28,13 @@
 #include "BCEfficiencyFitter.h"
 
 // ---------------------------------------------------------
-BCEfficiencyFitter::BCEfficiencyFitter()
-    : BCFitter()
-    , fHistogram1(0)
-    , fHistogram2(0)
-    , fFitFunction(0)
-    , fHistogramBinomial(0)
-    , fDataPointType(1)
-{
-    // set default options and values
-    fFlagIntegration = false;
-
-    // set MCMC for marginalization
-    SetMarginalizationMethod(BCIntegrate::kMargMetropolis);
-}
-
-// ---------------------------------------------------------
```

Comment:
Default ctors can come in very handy, especially when dealing with collections, dynamic allocation and ROOT's C++ reflection mechanisms. So even if the typical user won't need it, it might be better to keep it.